DigitalLearningSolutions.Data.Migrations/Scripts/UAR-831-CreateViewsForAdminUsersAndCandidatesTables-UP.sql

```diff
@@ -0,0 +1,80 @@
+﻿CREATE VIEW AdminUsers AS
+SELECT dbo.Users.ID                                      AS AdminID,
+       dbo.AdminAccounts.Login_deprecated                AS Login,
+       dbo.Users.PasswordHash                            AS Password,
+       dbo.AdminAccounts.CentreID,
+       dbo.AdminAccounts.IsCentreAdmin                   AS CentreAdmin,
+       dbo.AdminAccounts.ConfigAdmin_deprecated          AS ConfigAdmin,
+       dbo.AdminAccounts.IsReportsViewer                 AS SummaryReports,
+       dbo.AdminAccounts.IsSuperAdmin                    AS UserAdmin,
+       dbo.Users.FirstName                               AS Forename,
+       dbo.Users.LastName                                AS Surname,
+       dbo.AdminAccounts.Email,

```

Comment:
Should be the email on the new Users table?

Reply:
Yep.

Reply:
changed